config/environments/development.rb

```diff
@@ -26,10 +26,25 @@
     config.cache_store = :null_store
   end
 
-  # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  # Do care if the mailer can't send.
+  config.action_mailer.raise_delivery_errors = true
 
   config.action_mailer.perform_caching = false
+  config.action_mailer.default_url_options = { :host => 'localhost:3000' }
```

Comment:
Get rid of the hash rockets 🚀 

So this:

```ruby
config.action_mailer.default_url_options = { host: 'localhost:3000' }
```

Instead of this:

```ruby
config.action_mailer.default_url_options = { :host => 'localhost:3000' }
```